.github/ISSUE_TEMPLATE/breaking-change.yml

```
@@ -0,0 +1,107 @@
+name: ".NET breaking change"
+desription: Report a change in .NET that breaks something that worked in a previous version. Intended mostly for product-team use.
```

Hmmm.  When I try to view this on GitHub using "view file", it fails, saying that "description isn't valid". Any ideas?

---

This should fix https://github.com/dotnet/docs/pull/25161/files#r670465042.


```suggestion
description: Report a change in .NET that breaks something that worked in a previous version. Intended mostly for product-team use.
```